docs/sparc.client.services.rst

```diff
@@ -0,0 +1,21 @@
+sparc.client.services package
+=============================
+
+Submodules
+----------
+
+sparc.client.services.pennsieve module
+--------------------------------------
```

Comment:
Heading should be on another level?

Reply:
Resolving, as the discussion is in the previous section.

Reply:
best to leave unresolved until the issue is resolved, even though its the same issue mentioned in previous section...